frontend/types/env.d.ts

```diff
@@ -1,3 +1,6 @@
 declare module "@env" {
```

Comment:
i think we can actually delete this file and instead just use process.env values ([see docs here](https://docs.expo.dev/guides/environment-variables/))